web/packages/shared/components/UnifiedResources/ListView/ResourceListItem.tsx

```diff
@@ -183,51 +184,51 @@ export function ResourceListItem({
           <ResTypeIconBox mr={1}>
             <SecondaryIcon size={18} />
           </ResTypeIconBox>
-          <HoverTooltip
-            tipContent={resourceType}
-            css={`
-              // Required for text-overflow: ellipsis to work. This is because a flex child won't shrink unless
-              // its min-width is explicitly set.
-              min-width: 0;
-            `}
-            showOnlyOnOverflow
-          >
-            <Text fontSize="14px" fontWeight={300} color="text.slightlyMuted">
+          <HoverTooltip tipContent={resourceType} showOnlyOnOverflow>
+            <Text
+              fontSize="14px"
+              fontWeight={300}
+              color="text.slightlyMuted"
+              css={`
+                // Required for text-overflow: ellipsis to work. This is because a flex child won't shrink unless
+                // its min-width is explicitly set.
+                min-width: 0;
+              `}
+            >
               {resourceType}
             </Text>
           </HoverTooltip>
         </Flex>
 
         {/* address */}
-        <HoverTooltip
-          tipContent={addr}
-          showOnlyOnOverflow
-          css={`
-            grid-area: address;
-            overflow: hidden;
-            text-overflow: ellipsis;
-            white-space: nowrap;
-            // If the labels button isn't showing, let this column take up the extra space.
-            ${!showLabelsButton ? 'grid-column-end: labels-btn;' : ''}
-          `}
-        >
-          <Text fontSize="14px" fontWeight={300} color="text.muted">
+        <HoverTooltip tipContent={addr} showOnlyOnOverflow>
```

Comment:
I wasn't able to test this either. There are some apps with an overflow in the address, but the tooltip isn't shown. https://localhost:9002/?path=/story/shared-unifiedresources--list